frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -595,10 +595,18 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 		this.page.clear_fields();
 	}
 
-	refresh() {
+	refresh(have_filters_changed) {
 		this.toggle_message(true);
 		this.toggle_report(false);
 		let filters = this.get_filter_values(true);
+		let is_default_filters = this.filters
+			.map((filter) => {
+				return (
+					((!filter.default && !filter.value) || filter.default === filter.value) &&
+					!have_filters_changed
+				);
+			})
+			.every((res) => res === true);
```

Comment:
Can you explain this? with some examples
Also add some comments explaining it

Reply:
Yep, added explanation in the comments